Sources/SwiftParser/Declarations.swift

```diff
@@ -29,18 +29,25 @@ extension DeclarationModifier {
 }
 
 extension TokenConsumer {
+  mutating func atStartOfFreestandingMacroExpansion() -> Bool {
+    if !self.at(.pound) {
+      return false
+    }
+    if self.peek().rawTokenKind != .identifier && !self.peek().isLexerClassifiedKeyword {
```

Comment:
I would like to minimize the use of `isLexerClassifiedKeyword` in the hope that one day we can get rid of it. I think `!self.peek().rawTokenKind != .keyword` should work equally well here because you don’t care about all the pound keywords and the only lexemes that have a `RawTokenKind` of `keyword` are the lever classified keywords.